product-catalog/app/productcatalog/controllers/ProductDetailPageController.java

```diff
@@ -39,35 +38,33 @@
     private final ShippingMethodService shippingMethodService;
 
     @Inject
-    public ProductDetailPageController(final Configuration configuration, final ControllerDependency controllerDependency, final ProductProjectionService productService, final CategoryService categoryService, final ShippingMethodService shippingMethodService) {
+    public ProductDetailPageController(final ControllerDependency controllerDependency, final ProductProjectionService productService, final CategoryService categoryService, final ShippingMethodService shippingMethodService) {
```

Comment:
Because it is not used everywhere and also the module structure does not allow it.